benches/README.md

```diff
@@ -0,0 +1,71 @@
+# Miden VM Hash Functions 
+In Miden VM, we make use of different hash functions. Some of these are "traditional" hash functions, like `BLAKE3`, which are optimized for out of STARK performance, while others are algebraic hash functions, like `Rescue Prime`, and are more optimized for a better performance inside the STARK. In what follows, we benchmark several such hash functions and compare against other constructions that are used by other proving systems. More precisely, we benchmark:
```

Comment:
There seems to be an error in `which are optimized for out of`

Reply:
What is the error?

Reply:
It should be "out-of-STARK" performance. But maybe there is a better way to say it?